src/main/java/frc/robot/subsystems/endEffector/EndEffectorSubsystem.java

```diff
@@ -11,7 +11,7 @@
 
 public class EndEffectorSubsystem extends SubsystemBase {
 
-  EndEffectorIO endEffectorIO;
+  public EndEffectorIO endEffectorIO;
```

Comment:
Was talking about this earlier on 3/8, and making this public is an extremely hacky way to check for whether the coral is loaded or not.

Easiest and cleanest way to fix this tbh is to create a method within `EndEffectorSubsystem` that gets whether the coral is loaded